test/integration/Claims/dai-payouts.js

```diff
@@ -236,9 +235,9 @@ describe('DAI cover claim payouts', function () {
     assert(actualPayout.eq(expectedPayout), 'should have transfered the cover amount');
   });
 
-  it('[A1, status: 0, 7, 12, 13] CA accept, closed with closeClaim(), claim payout fails with status 12 and goes to status 13 after 60 retries', async function () {
+  it('[A1, status: 0, 7, -, 14] CA accept, closed with closeClaim(), claim payout fails, then succeeds', async function () {
```

Comment:
what does the - mean between 7 and 14?

Reply:
Claims can no longer have status 12 as the tx will fail. I've used `-` with the meaning that the status didn't change.